_posts/2019-10-21-starterkit-autumn-2019

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Starterkit Autumn 2019
+date: 2019-10-21
+categories: starterkit
+indico: https://indico.cern.ch/e/lhcb-starterkit-2019
+---
+
+
+Fasten your seatbelts once more! The LHCb Starterkit is taking off for the
+fifth time!
+This year's Starterkit will be held from the 21st to 25th of October 2019 at CERN.
+
+Registration is now open! You can find all necessary information, including the
+registration form on [the Indico page for the event][indico].
+
+The LHCb Starterkit is a yearly workshop designed to teach new students at LHCb
+the computing skills they need to become effective researchers. This year, we
+will once again teach the fundamentals of the LHCb software, as well as useful
+basics like Bash, Git and Python to 45 Masters and PhD students from LHCb.
+
+Additionally this year there will be 30 students from ALICE and 10 students from SHiP collaborations joining for the
```

Comment:
@marthaisabelhilton Do you know if there will be any SHiP/ALICE involvement?

Reply:
@chrisburr Yes the plan is as usual to have SHiP and ALICE there. I have met with the organisers from both experiments a few weeks ago and they planned to open registration then. I will get in touch with them to see how things are going. We have three groups for the first two days as usual with ALICE contributing some helpers and teachers.

Reply:
Wonderful! 👍